trajectory_tracker/src/trajectory_tracker.cpp

```diff
@@ -405,16 +405,23 @@ void TrackerNode::control()
         trajectory_tracker::timeOptimalControl(-remain_local * sign_vel_, acc_[0]),
         vel_[0], acc_[0], dt);
 
-    const float wref = std::abs(v_lim_.get()) * curv;
+    const float wref_raw = std::abs(v_lim_.get()) * curv;
+    float wref = wref_raw;
 
-    if (limit_vel_by_avel_ && std::abs(wref) > vel_[1])
+    if (limit_vel_by_avel_ && std::abs(wref_raw) > vel_[1])
```

Comment:
keep curvature by decreasing both angular vel and linear vel if angular vel is larger than the limit